addons/links/README.md

```diff
@@ -16,45 +16,100 @@ This addon works with Storybook for:
 ## Getting Started
 
 Install this addon by adding the `@storybook/addon-links` dependency:
+
 ```sh
 yarn add @storybook/addon-links
 ```
 
-First configure it as an addon by adding it to your addons.js file (located in the Storybook config directory).
+Then configure it as an addon by adding it to your addons.js file (located in the Storybook config directory).
 
 ```js
 import '@storybook/addon-links/register';
 ```
 
-Then you can import `linkTo` in your stories and use like this:
+## LinkTo component
```

Comment:
This is really react-specific.

Reading this and the issue this came from, adding this isn't really required right?
Can we do without it?

I'm mostly on-board with the `hrefTo` function. Although I kind of struggle to understand the use-case.

Reply:
> I'm mostly on-board with the `hrefTo` function. Although I kind of struggle to understand the use-case.

The use-case is when you have pure presentational components which take URLs as props and want them to link to other parts of your story.

Reply:
Yes it is react-specific, but it allows creating links which can open the story in new tab as well as in the same tab (without reloading the page). I can adjust `README` to indicate that this feature is react only. Users could implement similar components for other frameworks by combining `linkTo` and `hrefTo`

Reply:
Yes, `linkTo` already works cross-platform, I'm expecting `hrefTo` to work cross platform too.

Please update to readme to reflect this, and move the react-specific to the bottom of the 3, so the cross-platform stuff is at the top.

Reply:
I've made the adjustments